src/source_map_utils.ts

```diff
@@ -135,8 +135,8 @@ export class DefaultSourceMapper implements SourceMapper {
 
   constructor(private fileName: string) {
     this.sourceMap.addMapping({
-      original: {line: 1, column: 1},
-      generated: {line: 1, column: 1},
+      original: {line: 1, column: 0},
```

Comment:
I don't understand this change, but maybe I'm just mislead by the commit message.

In TypeScript (as in: `tsc` and the `typescript` node module & APIs), line and column are zero based. I cannot find an authoritative source on source maps (it'd be nice if you could link to one here), but at least `sourcemap.ts` in the TS code base seems to assume 1-based indexing, whereas the Closure Compiler assumes 0-based for the most recent format. https://github.com/google/closure-compiler/blob/master/src/com/google/debugging/sourcemap/FilePosition.java#L38

Maybe that's the problem you're fixing here?

In any case, please add a comment explaining what's going on, with a pointer to exactly what bits are breaking, what you are fixing, and a test case.